pubspec.yaml

```diff
@@ -35,6 +35,9 @@ dependencies:
   pull_to_refresh: ^1.6.1
   feather_icons_flutter: ^4.7.4
   shared_preferences: ^0.5.12
+  flutter_bloc: ^6.1.0
+  equatable: ^1.2.5
+  meta: ^1.2.3
```

Comment:
I think these packages aren't being used in the app.

Reply:
Oh yes, I forgot to remove them while trying the bloc pattern. I have removed them for now.